src/lib/list/selection-list.ts

```diff
@@ -132,6 +132,12 @@ export class MdListOption extends _MdListOptionMixinBase
     super();
   }
 
+  ngOnInit() {
```

Comment:
Can you add the `OnInit` interface to the class?

Reply:
Done